docs/application/dotnet/get-started/wearable/first-app.md

```diff
@@ -71,113 +60,67 @@ To create a new Tizen .NET project:
 
-For this example application, the default manifest is sufficient. If you
-want to make any changes in the application, such as changing the
-application icon or installing resources that are used by the
-application at runtime, see [Packaging Your Application](#packaging-your-application).
+For this example application, the default manifest is sufficient. If you want to make any changes in the application, such as changing the application icon or installing resources that are used by the application at runtime, see [Packaging Your Application](#packaging-your-application).
 
 After you have built the application, deploy and run it.
 
 ## Deploying and Running Your Application
 
-To run the application, you must first deploy it to the target: either a
-device or an emulator. Deploying means transferring the package file
-(`.tpk`) to the target and invoking the Tizen Package Manager to install
-it.
+To run the application, you must first deploy it to the target: either a device or an emulator. Deploying means transferring the package file (`.tpk`) to the target and invoking the Tizen Package Manager to install it.
 
 To deploy and run the application on the emulator:
 
-1.  In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen
-    Emulator Manager**.
+1.  In the Visual Studio menu, select **Tools \> Tizen \> Tizen Emulator Manager**.
 
     Alternatively, click **Launch Tizen Emulator** in the Visual Studio toolbar to launch the Tizen Emulator Manager.
 
     ![Launch Tizen Emulator](media/cs_launch_tizen_emu.png)
 
-2. In the Emulator Manager, select an emulator from the list and click
-    **Launch**.
+2. In the Emulator Manager, select an emulator from the list and click **Launch**.
 
-    If no applicable emulator instance exists, [create
-    one](../../../vstools/tools/emulator-manager.md#create).
+    If no applicable emulator instance exists, [create one](../../../vstools/tools/emulator-manager.md#create).
 
-    ![Tizen Emulator
-    Manager](media/cs_first_building_emulator_manager.png)
+    ![Tizen Emulator Manager](media/cs_first_building_emulator_manager.png)
 
-3. Once you launch an emulator instance, you can deploy the application
-    by clicking the emulator instance in the Visual Studio toolbar.
+3. Once you launch an emulator instance, you can deploy the application by clicking the emulator instance in the Visual Studio toolbar.
 
     In the Visual Studio toolbar, you can select the target from the drop-down list to change the deployment target.
 
-    ![Tizen Emulator Manager](media/vs_emulator_launch.png)
```

Comment:
Question to this image update.

In the md file, vs_emulator_launch.png file is excluded so it is not in use now.
However, the file itself is also update with this PR, 
 - from the image selecting the emulator : https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/get-started/wearable/media/vs_emulator_launch.png
 - to the display example : https://github.com/Samsung/tizen-docs/blob/b4a9d628ae50122386f12f92f3858dedc9152ee8/docs/application/dotnet/get-started/wearable/media/vs_emulator_launch.png

I'm not sure with this image update. Is there any reason?

Reply:
The update image shows all of the things.

Reply:
Yes, that is the reason of my question.

Why the old image, vs_emulator_launch.png is also updated?
vs_emulator_launch.png is useless now,
so can't we just delete this image instead of updating it?